packages/ibm-products-styles/src/components/ConditionBuilder/styles/_conditionBuilderCondition.scss

```diff
@@ -14,12 +16,22 @@ $block-class: #{c4p-settings.$pkg-prefix}--condition-builder;
   display: inline-flex;
 }
 
-[role='row']:focus {
-  @include focus-outline.focus-outline('outline');
+// [role='row']:focus {
+//   @include focus-outline.focus-outline('outline');
```

Comment:
Better remove commented code all across the files.

Reply:
resolved as advised